mlcvs/core/loss/elbo.py

```diff
@@ -68,6 +70,6 @@ def elbo_gaussians_loss(
         kl = (kl * weights).sum()
```

Comment:
@andrrizzi, what is the shape of kl here? 
I was noticing that in mse_loss we unsqueeze the weights to (n_batch,1), while here we apply them directly.